app/routes.js

```diff
@@ -0,0 +1,31 @@
+var fs = require('fs');
+	express = require('express');
```

Comment:
Pls lint your code for the last time! This is not allowed. All variables must be individually defined. You not even using a comma so this is going to fail anyway